### DISCUSSION TITLE
@this.previousElementSibling

### DISCUSSION THREAD

**Discussion by anjanesh** [alpinejs/alpine#3579] [1. Help] [answered]
How do I get something like this work ?

```
<div x-show="@this.previousElementSibling.previousElementSibling.checked">X</div>
```

**Comment by ekwoka** [comment, marked as answer]
1. `$el.previousElementSibling.checked`
2. That won't be reactive, you can instead model the element to state and use the state in x-show.
3. The peer (`~`) selector in css can also do this without needing to involve any js
